.github/workflows/build-kubespray-image.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: Build kubespray images
+
+on:
+  push:
+    branches: [master, release-.*]
+  release:
+    branches: [master, release-.*]
+
+env:
+  IMAGE_REPO: "kubespray"
+  IMAGE_NAME: "kubespray"
+  IMAGE_REGISTRY: "hub.k8s.li"
```

Comment:
Don't forget to change that, as long as this is set we can't do anything

Reply:
done